### PR TITLE
[WebAPI] Switch WebGL testing to 1.0.3 conformance tests

### PR DESCRIPTION
**Pull request by renchenglei**
Impacted tests(approved): new 678, update 0, delete 1587
 Unit test platform: Crosswalk Project for Android 19.49.514.0
 Unit test result summary: pass 674, fail 4, block 0

 BUG=XWALK-6780

### CODE REVIEW

**Comment by renchenglei** [comment]
@zqzhang , PTAL.
 Thanks a lot!

File: webapi/webapi-webgl-khronos-tests/COPYING [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2012 Intel Corporation.
```
Comment (zqzhang) [inline webapi/webapi-webgl-khronos-tests/COPYING]: s/2012/2016/

File: webapi/webapi-webgl-khronos-tests/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# WebGL Test Suite
+
+## Introduction
+
+This test suite is for checking compliance with WebGL (Khronos) specification:
+* https://www.khronos.org/registry/webgl/specs/1.0/
+
+## Authors
+
+* Zhang, Zhiqiang <zhiqiang.zhang@intel.com>
```
Comment (zqzhang) [inline webapi/webapi-webgl-khronos-tests/README.md]: Please remove me from here.

File: webapi/webapi-webgl-khronos-tests/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+* Ren, Chenglei <chenglei.ren@intel.com>
+
+## LICENSE
+
+Copyright (c) 2013 Intel Corporation.
```
Comment (zqzhang) [inline webapi/webapi-webgl-khronos-tests/README.md]: s/2013/2016/

**Comment by zqzhang** [comment]
@chunywang @haoxli could you please help verify this test suite? Many thanks.

**Comment by zqzhang** [comment]
@qiuzhong @BruceDai we will use this test suite `webapi-webgl-khronos-tests` to replace `tct-webgl-nonw3c-tests` after we evaluate it. Please closely monitor this pull request and update your scripts if needed. Thanks.

**Comment by chunywang** [comment]
@zqzhang  I can't am the patch as trailing whitespace. @renchenglei

**Comment by chunywang** [comment]
Impacted tests(approved): new 0, update 678, delete 0
Unit test platform: Crosswalk Project for Android 19.49.514.0
Unit test result summary: pass 657, fail 21, block 0

**Comment by renchenglei** [comment]
@chunywang ,Thanks a lot!

File: tools/testsync/testsync.py [hunk 4]
```diff
@@ -43,7 +43,7 @@
 
 SCRIPT_PATH = os.path.realpath(__file__)
 CONST_PATH = os.path.dirname(SCRIPT_PATH)
-DEST_DIR = "/work/webapi/ww/ww43/crosswalk-test-suite/webapi/tct-webgl-nonw3c-tests/webgl/khronos"
+DEST_DIR = "/work/webapi/ww/ww43/crosswalk-test-suite/webapi/webapi-webgl-khronos-tests/webgl/khronos"
 ORIGIN_DIR = "/work/webapi/ww/ww43/KhronosGroup/WebGL/conformance-suites/1.0.2"
```
Comment (zqzhang) [inline tools/testsync/testsync.py]: s/1.0.2/1.0.3/

**Pull request merged**